src/engraving/dom/score.h

```diff
@@ -611,7 +611,7 @@ class Score : public EngravingObject, public muse::Injectable
     Selection& selection() { return m_selection; }
     SelectionFilter& selectionFilter() { return m_selectionFilter; }
     void setSelection(const Selection& s);
-
+    void selectElementsWithSameTypeOnSegment(mu::engraving::ElementType elementType, mu::engraving::Segment* segment);
```

Comment:
It would be ideal to put this method below `select(const std::vector&……` and preserve the empty line here, so that the methods are properly grouped

Reply:
Moved!